backend/src/firemerge/statement/parser.py

```
@@ -115,7 +118,12 @@ def _parse_rows(
                 and (doc_number := row[join_col.index])
                 and (join_row := join_rows.get(doc_number))
             ):
-                transaction = self._parse_row(row, join_row)
+                if duplicates and doc_number in duplicates:
+                    logger.warning(f"Duplicate doc number: {doc_number}")
```

Pylint (4.0.5)

[warning] Use lazy % formatting in logging functions

(W1203)